wrappers/ortools.rb

```diff
@@ -606,6 +627,7 @@ def parse_output(vrp, _services, points, _matrix_indices, _cost, _iterations, ou
           earliest_start = route_start
           {
             vehicle_id: vehicle.id,
+            costs: route_costs,
             activities: route['activities'].collect.with_index{ |activity, activity_index|
```

Comment:
It is better if you can replace all these `A['X']` with `A.X` if protobuf has the functions otherwise you break the uniformity which will bite us in the back in the future.